### PR TITLE
Fix bugs and merge to fix the functionality of production

### PR DESCRIPTION
**Pull request by TrNgTien**

### CODE REVIEW

File: src/store/actions/searching-project-user/project/project.js [hunk 1]
```diff
@@ -93,23 +93,29 @@ export const _onSubmitFormAnswers = (answers, organizationId, projectId) => {
...
     try {
       await firebase_db
         .collection("organization")
-        .doc(`${organizationId}`)
+        .doc(`${orgId}`)
         .collection("projects")
         .doc(`${projectId}`)
         .collection("formSubmission")
-        .doc(`${studentUID}`)
+        // .doc(`${studentUID}`)
```
Comment (kenpham4real) [inline src/store/actions/searching-project-user/project/project.js:109]: Why did you delete this line? It is required to push the data to the exact student @TrNgTien

File: src/store/actions/searching-project-user/project/project.js [hunk 2]
```diff
@@ -93,23 +93,29 @@ export const _onSubmitFormAnswers = (answers, organizationId, projectId) => {
...
-                studentName: studentName,
-                studentEmail: studentEmail,
-                studentUID: studentUID 
-            }
+            answers: answerInput,
+            // studentInfo: {
+            //     studentName: studentName,
+            //     studentEmail: studentEmail,
+            //     studentUID: studentUID 
+            // }
```
Comment (kenpham4real) [inline src/store/actions/searching-project-user/project/project.js:118]: Can't see the reason why this was deleted. @TrNgTien

**Review by kenpham4real — CHANGES_REQUESTED** [review]

File: src/store/actions/searching-project-user/project/project.js (hunk 2 above)
Comment (TrNgTien) [reply src/store/actions/searching-project-user/project/project.js:118]: Plz check your messenger i told u the detail in messenger
.doc(${studentUID}) is the problem why we can push the data of answers to Firebase.Becz the getState() can not get the State from reducer so it was wrong
